src/pages/index.jsx

```diff
@@ -1,21 +1,85 @@
+import communityorganizers from '../images/icons/communityorganizers.svg'
+import datascientists from '../images/icons/datascientists.svg'
+import designers from '../images/icons/designers.svg'
+import developers from '../images/icons/developers.svg'
+import productmanagers from '../images/icons/productmanagers.svg'
+import projectmanagers from '../images/icons/projectmanagers.svg'
+import socialworkers from '../images/icons/socialworkers.svg'
+
+const WorkWithUs = styled.section`
+  margin-bottom: 4rem;
+`
 
 const IndexPage = () => {
+  const icons = [
+    { img: developers, title: 'Developers' },
+    { img: designers, title: 'Designers' },
+    { img: datascientists, title: 'Data Scientists' },
+    { img: projectmanagers, title: 'Project Managers' },
+    { img: productmanagers, title: 'Product Managers' },
+    { img: communityorganizers, title: 'Community Organizers' },
+    { img: socialworkers, title: 'Social Workers' },
+    { img: anyone, title: 'Anyone' },
+  ]
+
   return (
     <Layout>
-      <h2>homepage</h2>
       <Hero />
-      <Card />
-      <Button text={'Submit'} small arrow />
-      <Button text={'Submit'} arrow />
-      <Button text={'Button'} secondary small />
-      <Button text={'Button'} secondary />
-      <Button text={'Link'} textBtn arrow link small />
-      <Button text={'Join Us on Slack'} textBtn arrow link />
+      <WorkWithUs>
+        <Heading
+          headingText="Work With Us"
+          description="Whoever you are, there is a place for you here! Join us for co-working, project, service, and social events."
+        />
+        <IconGrid icons={icons} />
+      </WorkWithUs>
+
+      <ContentSection
+        background={'var(--blizzard-blue)'}
+        eyebrowText={'Organizations'}
+        mainHeading={
+          'Are you a Chicagoland nonprofit that needs some tech help?'
+        }
+        content={
+          "Have an idea how to make your community better? You've come to the right place. Either contact us or come to one of our meetups to get the conversation started."
+        }
+        linkText={'Learn More'}
+        linkUrl={'/'}
+        imgUrl={Organizations}
+        imageDescription={'Organization receiving help from Code for Chicago'}
+      />
+      <ContentSection
+        background={'var(--light-grey)'}
+        eyebrowText={'Volunteers'}
+        mainHeading={
+          'Looking to use your skills for the greater good and build your resume?'
+        }
+        content={
+          'Maybe you’re looking to pick up a new tech skill. You too, have come to the right place. Sign up for our meetups or join us on Slack.'
+        }
+        linkText={'Join us on Slack'}
+        linkUrl={
+          'https://join.slack.com/t/c4chi/shared_invite/zt-1mvwfme5d-0FtxC_4ZcGW9StKgfBvToQ'
+        }
+        imgUrl={Volunteers}
+        imageDescription={'Organization receiving help from Code for Chicago'}
+        imgLeft
+      />
+      <Heading
```

Comment:
Do we need a different button here linking the Meetup Group? This one is bigger than the component sizes.